_posts/2016-12-30-end-of-year-fourth-quarter-updates-for-municipal-boundaries.md

```diff
@@ -18,20 +18,20 @@ tags:
 
 ---
 
-Once again we have reached the end of the year, and all updates for the Municipal Boundaries are available online and listed below.
+Once again we have reached the end of the year, and all updates for the Municipal Boundaries are available for [download](https://gis.utah.gov/data/boundaries/citycountystate/) and listed below.
```

Comment:
please do not link to gis.utah.gov directly. [There are instructions](https://github.com/agrc/gis.utah.gov#link-to-page) in the readme if you need a reminder.